app/assets/javascripts/search.js

```diff
@@ -1,31 +1,145 @@
 $(document).ready(function() {
 
-  /* open the Adv Search UI immediately if param set */
+  /*************************************************************************************************/
+  /* ** Search Form ** */
```

Comment:
oops! just left some JS comments on https://github.com/18F/C2/pull/1117 bc I didn't realize it was a branch off of this. Sorry! reference that for my js feedback :green_apple: